### PR TITLE
tw-asmft-430

### PR DESCRIPTION
**Pull request by TrentonWalkup**
This is the shell for adonis ios app.

### CODE REVIEW

**Comment by mikepoe** [comment]
LGTM, is this IOS only or android too?  I noticed some android stuff in there

**Comment by TrentonWalkup** [comment]
just IOS but I did the test_object_config at the same time

File: demo/wdio.testobject.ios.conf.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+var secure = require('./secret.js')
+
+ exports.config = {
+    protocol: 'https',
+    //host: 'eu1.appium.testobject.com', this is for server in europe. do not delete
```
Comment (jeremycrussell) [inline demo/wdio.testobject.ios.conf.js]: rm comment?

File: demo/wdio.testobject.ios.conf.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    connectionRetryTimeout: 10 * 600000,
+    connectionRetryCount: 3,
+    framework: 'mocha',
+    mochaOpts: {
+        ui: 'bdd',
+        timeout: 2000000
+    },
+}
+
+// exports.config = config
```
Comment (jeremycrussell) [inline demo/wdio.testobject.ios.conf.js]: rm?

**Comment by Gregorma** [comment]
LGTM

**Comment by Gregorma** [comment]
LGTM

**Pull request merged**